extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -666,10 +699,15 @@ export class ConsentUI {
     }
     setImportantStyles(this.parent_, {
       transform: `translate3d(0px, calc(100% - ${this.initialHeight_}), 0px)`,
+      '--modal-height': `${this.initialHeight_}`,
```

Comment:
nit: scope namespace
```suggestion
      '--i-amp-modal-height': `${this.initialHeight_}`,
```